src/components/gcds-checkbox/gcds-checkbox.css

```diff
@@ -0,0 +1,118 @@
+:host {
+  .gcds-checkbox-wrapper {
+    max-width: 90%;
+    font-family: var(--gcds-font-families-body), sans-serif;
+    font-weight: var(--gcds-font-weights-regular);
+    font-size: var(--gcds-font-sizes-paragraph);
+    line-height: var(--gcds-line-heights-paragraph);
+    color: var(--gcds-checkbox-default-text);
+    margin: 0;
+    padding: 0;
+    border: 0;
+    transition: color ease-in-out 0.15s;
+
+    position: relative;
+    margin-bottom: 20px;
+    margin-top: 28px;
+
+    &:focus-within {
+      color: var(--gcds-checkbox-focus);
+    }
+
+    &.gcds-disabled {
+      color: var(--gcds-checkbox-disabled-text);
+
+      input[type=checkbox]+gcds-label {
+        color: var(--gcds-checkbox-disabled-text);
+        &:before {
+          border-color: var(--gcds-checkbox-disabled-border);
+          background-color: var(--gcds-color-grayscale-100);
+          cursor: not-allowed;
+        }
+        &:after {
+          cursor: not-allowed;
+        }
+      }
+    }
+
+    &.gcds-error:not(:focus-within) {
```

Comment:
Same comment as for the disabled - the other form fields are currently using the `error` class instead of -`1gcds-error`. Can we update those as well to stay consistent within the form fields?